src/pudl/analysis/record_linkage/eia_ferc1_splink_model_config.py

```diff
@@ -0,0 +1,93 @@
+"""The model parameters for the FERC1 to EIA splink record linkage model.
```

Comment:
Does it make sense to have these model parameter definitions hang out in this separate module?

Reply:
Ah good idea. I'll make an issue in SEC-EIA land so that we can do this for the next splink model we create.